### ISSUE TITLE
Lut3d module not showing up

### ISSUE THREAD

**Issue by Revheim** [darktable-org/darktable#8096]
I have built darktable according to the instructions here: https://raw.githubusercontent.com/darktable-org/darktable/master/packaging/windows/BUILD.txt
I have renamed libopencv_core4xx and libopencv_videoio4xx to libopencv_core440 and libopencv_videoio440 as per the instructions.
The module gets built, I can see it at: "C:\Program Files\darktable\lib\darktable\plugins\liblut3d.dll"
But the lut3d module do not show up among my available modules.

 - Darktable Version: 3.5.0+947~g47c0146e
 - OS: windows 10 64 bit

**Comment by Mark-64** [comment]
I just built DT git master in Windows 10 and LUT3D works for me.
Do you have other versions of libopencv_core4xx and libopencv_videoio4xx in the program folder ?
Or try to unistall DT, delete "C:\Program Files\darktable", reinstall and rename libs

**Comment by Revheim** [comment]
I just did a complete search for the libs on my system (phew, that took a while), but no, these are the only ones. 
I have also uninstalled, removed the darktable install-directory, and reinstalled but now I get an error-message stating the application was unable to start correctly :/

So, I uninstalled again, deleted everything inside the build-directory, and recompiled, again... and again I got the same error-message.

Next I removed the whole darktable src directory, and did a fresh git clone, compiled packaged and installed, renamed the two libs and voila, now it works.

I find all of this very strange as I've had no such problems before, and I have not changed anything in any source-files.
Well, at least I have a working build again.
